diff --git a/node_modules/git-rev-sync/index.js b/node_modules/git-rev-sync/index.js
index e0d0eea..172d0be 100644
--- a/node_modules/git-rev-sync/index.js
+++ b/node_modules/git-rev-sync/index.js
@@ -93,8 +93,8 @@ function long(dir) {
   }
 
   var gitDir = _getGitDirectory(dir);
-  var gitRootDir = gitDir.indexOf('.git/worktrees/') > 0 ?
-    gitDir.replace(/\.git\/worktrees\/.+$/, '.git') :
+  var gitRootDir = gitDir.indexOf('.git/worktrees/') > 0 || gitDir.indexOf('.git\\worktrees\\') > 0 ?
+    gitDir.replace(/\.git[\/\\]worktrees[\/\\].+$/, '.git') :
     gitDir;
   var refsFilePath = path.resolve(gitRootDir, 'refs', 'heads', b);
   var ref;
